src/scripts/main.js

```diff
@@ -1,3 +1,398 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const tableBody = document.querySelector('tbody');
+const tableHeader = document.querySelector('thead');
+
+const numbers = '/^[0-9]+$/';
+const letters = '[^a-zA-Z -]';
+
+let clickCounter = 0;
+let checkElement;
+
+function salaryToNumber(salary) {
+  return +(salary.split('$').join('').split(',').join(''));
+}
+
+function stringToSalary(string) {
+  return '$' + string.toString().replace(/(\d)(?=(\d\d\d)+(?!\d))/g, '$1,');
+}
+
+tableHeader.addEventListener('click', (e) => {
+  const element = e.target.closest('th');
+  const cols = table.querySelectorAll('thead th');
+  const index = [...cols].indexOf(element);
+  const sortedRows = [...tableBody.querySelectorAll('tr')];
+
+  clickCounter++;
+
+  if (checkElement !== element) {
+    clickCounter = 1;
+  }
+
+  checkElement = element;
+
+  sortedRows.sort((a, b) => {
+    const elementA = a.children[index].innerText;
+    const elementB = b.children[index].innerText;
+
+    if (element.innerText.toLowerCase() === 'salary') {
+      return (clickCounter % 2 === 1)
+        ? salaryToNumber(elementA) - salaryToNumber(elementB)
+        : salaryToNumber(elementB) - salaryToNumber(elementA);
+    }
+
+    if (element) {
+      return (clickCounter % 2 === 1)
+        ? elementA.localeCompare(elementB)
+        : elementB.localeCompare(elementA);
+    }
+  });
```

Comment:
maybe move sorting logic to separate method

Reply:
Really... Thank you, Artem! I'll do so! ☺